app/medInria/medFilteringWorkspace.cpp

```diff
@@ -44,6 +44,7 @@ medFilteringWorkspace::medFilteringWorkspace(QWidget *parent): medAbstractWorksp
 {
     d->filteringToolBox = new medFilteringSelectorToolBox(parent);
     connect(d->filteringToolBox,SIGNAL(processFinished()),this,SLOT(onProcessSuccess()));
+    d->filteringToolBox->setWorkspace(this);
```

Comment:
Thanks Flo but you created this enhancement issue after :D

I agree with your issue if you want to do it :)